sechub-pds/src/main/resources/application-pds_localserver.yml

```diff
@@ -12,8 +12,8 @@ server:
     key-alias:
        tomcat
   port: 8444
-sechub:
-  pds:
+
+pds:
     server:
```

Comment:
Same as above: please choose one amount of blanks for indentation in the file (e.g. 2 blanks).